fsw/src/crypto.c

```diff
@@ -3410,57 +3427,60 @@ int32 Crypto_TC_ProcessSecurity( char* ingest, int* len_ingest,TC_t* tc_sdls_pro
             OS_printf("\tsa[%d].iv[%d] = 0x%02x \n", tc_sdls_processed_frame->tc_sec_header.spi, sa_ptr->shivf_len-1, sa_ptr->iv[sa_ptr->shivf_len-1]);
         #endif
 
-        // Check IV is in ARCW
-        if (Crypto_window(tc_sdls_processed_frame->tc_sec_header.iv, sa_ptr->iv, sa_ptr->shivf_len,
-            sa_ptr->arcw[sa_ptr->arcw_len-1]) != OS_SUCCESS )
+        if ( crypto_config->ignore_anti_replay==TC_IGNORE_ANTI_REPLAY_FALSE )
         {
-            report.af = 1;
```

Comment:
No changes here actually... I've just surrounded this by a giant if block based on config.

Reply:
and git doesn't show the diff particularly well.